README.md

```diff
@@ -210,4 +233,14 @@ request
 |
 |--|---|----|------>
    1   3    10     #Replicas
-```
\ No newline at end of file
+```
+
+### Custom Resource Definition
+
+#### Types
+
+See [here](https://github.com/gardener/hvpa-controller/blob/a336e86111bbd5ec87f5a749003ba39b4735261e/api/v1alpha1/hvpa_types.go#L28-L305).
```

Comment:
s/a336e86111bbd5ec87f5a749003ba39b4735261e/master/

Reply:
Thought `master` would be moving target. Hence, the permalink. WDYT?